### DISCUSSION TITLE
[Help]: Full-screen slides example

### DISCUSSION THREAD

**Discussion by rolanday** [davidjerleke/embla-carousel#702] [Help] [answered]
### Summary

Hello,

Are there any examples showing how to configure Embla for fullscreen (with and without thumbs)? I am envisioing a fullscreen carousel with thumbs at the bottom, header at top, and slides taking up all remaining space between for mobile device, and media query setting limits for how large container can grow.

All of the examples I've found configure Embla `--slide-height` for a fixed value (and setting to 100% does not work). I've managed to workaround with my own CSS vars, but it's sending me down a rabbit hole (e.g., what works in chrome doesn't render the same as in Firefox, unexpected Embla behaviors, etc.), so feel like I'm heading down the wrong path, and I suspect this is a common use case.

Any help much appreciated!!

Thanks,

**Comment by davidjerleke** [comment]
Hi @rolanday,

Which Embla Carousel package are you using (i.e. React, Vanilla…)?

**Comment by rolanday** [comment]
Hi David, appreciate you checking back. I am using React.

**Reply by davidjerleke** [reply, marked as answer]
@rolanday here's a [CodeSandbox](https://codesandbox.io/p/sandbox/embla-carousel-full-screen-react-pkprcq) with the full screen API.

> [!NOTE]  
> Here's [browser support](https://caniuse.com/fullscreen) for the [full screen API](https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API).

Best,
David

**Reply by davidjerleke** [reply]
@rolanday, have you had the chance to [check my example](https://github.com/davidjerleke/embla-carousel/discussions/702#discussioncomment-8514242) out?

**Reply by rolanday** [reply]
Hi @davidjerleke , just say this and sandbox demo looks great. Will give it a spin on my end this week. Thank you!!

**Reply by davidjerleke** [reply]
@rolanday sounds good. Let me know when you’ve done that and if it’s what you want to achieve.

**Reply by davidjerleke** [reply]
@rolanday have you had the chance to try the CodeSandbox out?

**Reply by rolanday** [reply]
Hi @davidjerleke , works like a charm! Thank you for your time and attention -- super appreciated!!
